### ISSUE TITLE
Automatic suggestions in the address fields in www.gmail.com compose message are not read by NVDA

### ISSUE THREAD

**Issue by sumandogra** [nvaccess/nvda#5676]
NVDA remains silent and does not give any info bout the automatic suggestions in the address fields in the compose message of www.gmail.com.
Steps to reproduce:
1.  Open www.gmail.com.
2.  Login to your account.
3.  Press ALT+C to open compose message window.
4.  Go to the address edit field, To, Cc.
5.  Type in a few characters for the address already resent in the contacts list.
6.  No automatic suggestions are read.
7.  TAB to input and address. No information is read by NVDA

**Comment by jcsteh** [comment]
What browser? Works fine for me in Firefox.

**Comment by sumandogra** [comment]
It does not work in internet explorer.

**Comment by nvaccessAuto** [comment]
Incubated in b7185862b21546faa09738100181de036ddb0540.

**Comment by nvaccessAuto** [comment]
Incubated in 47daf9a1d527fcfe15941bfb8fe929e1b083101c.

**Comment by nvaccessAuto** [comment]
Incubated in c078fccc7d303f66127c53019a20407911fdc7ee.

**Comment by jcsteh** [comment]
@sumandogra: What version of Internet Explorer were you testing with? While we did fix a problem with NVDA switching to browse mode incorrectly when address suggestions appear, even without this fix, suggestions still get read for us. Could you please re-test this with both master and next branches? If you can still reproduce this, a log file at level input/output would be useful.

**Comment by jcsteh** [comment]
This fix addresses one issue, but we couldn't reproduce the original report, so leaving open for now pending a response from @@sumandogra.

**Comment by bhavyashah** [comment]
@sumandogra Could you please respond to https://github.com/nvaccess/nvda/issues/5676#issuecomment-209245829?

**Comment by Adriani90** [comment]
I cannot reproduce this issue in IE11 and the author seems not to be available anymore. Closing this issue for now as works for me. If someone reports it again we can reopen it. Thanks.
